website/docs/examples/multi_client_geoimage/README.md

```diff
@@ -0,0 +1,236 @@
+---
+id: multi-client-geoimage
+title: Multi Client GeoImage
+---
+
+### [Link to `multi_client_geoimage/main.py`](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/examples/multi_client_geoimage/main.py)
+
+This example shows how to leverage a multi client application to create a geoimage
+from the `oak` and `gps` services. We show how to implement a simple technique to
+synchronize the images and the gps data.
+
```

Comment:
You could put a recommendation with a link to go through the multi client subscriber example first